src/style.css

```
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  margin: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+}
+
+.product__card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: white;
+  
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  padding-top: 32px;
+  margin: 0 auto;
+}
+
+.card__name {
+  width: 166px;
+  height: 36px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  padding-top: 40px;
+  margin: 0 16px;
+}
+
+.card__code {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+  padding-top: 4px;
+  margin-left: 16px;
+
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 16px;
+  margin: 0 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--4 > :nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__review-number {
+  padding-top: 2px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  padding-top: 24px;
+  padding-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px;
+}
+
+.card__price-cost {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070 ;
+}
+
+.card__price-value {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  height: 40px;
+  width: 166px;
+  border-radius: 5px;
+  margin: 0 auto;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+  background-color: #00acdc;
+}
+
+.card__button:hover {
+  width: 166px;
+  height: 40px;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  background-color: #fff;
+}

```

remove default styles that card__button has
also add border to card__button

```suggestion
.card__button:hover {
  color: #00acdc;
  background-color: #fff;
}

```